src/interface/src/app/standalone/planning-areas/planning-areas.component.html

```diff
@@ -118,13 +119,24 @@ <h3>Home</h3>
 <ng-template #emptyState>
   <section class="empty-state">
     <mat-icon class="material-symbols-outlined">create_new_folder</mat-icon>
-    <div class="empty-state__title">No Planning Areas Yet</div>
-    <div>
-      Click "Explore" to start creating your planning areas. Not ready to create
-      new plans?
-    </div>
-    <div>
-      You can explore the mapping tool without creating a planning area.
-    </div>
+
+    <ng-container *ngIf="hasFilters$ | async; else noData">
+      <div class="empty-state__title">No Planning Areas Found</div>
+      <div>No planning areas match the search criteria provided.</div>
+      <div>
+        Please verify spelling, use alternative keywords, or broaden the search
```

Comment:
Is this hasFilters state on the Figma anywhere? I'm wondering if it ought to have the same icon, or maybe something else to signify that nothing was found.

Reply:
its not but I pinged Julie and Jordan on the story to confirm